jobs/env/ci-kubernetes-e2e-gce-large-performance.env

```diff
@@ -22,3 +22,5 @@ TEST_CLUSTER_RESYNC_PERIOD=--min-resync-period=12h
 TEST_CLUSTER_DELETE_COLLECTION_WORKERS=--delete-collection-workers=16
 # Switch off image puller to avoid bloating node objects (#44701)
 PREPULL_E2E_IMAGES=false
+# Turn off advanced audit logging to simulate production
```

Comment:
If we decide to move forward with this PR, can you file an issue to re-enable it in 1.9, and add a TODO here?

Reply:
yes, I'd open an issue

I'd probably put the envvar in config-default.sh in the main repo with a TODO to update these jobs